catalog-info.yaml

```
@@ -24,7 +24,7 @@ spec:
       repository: elastic/pre-commit-buildkite-plugin
       pipeline_file: ".buildkite/pipeline.yml"
       teams:
-        release-eng:
-          access_level: MANAGE_BUILD_AND_READ
+        release-eng: {}
+        platform-dev-flow: {}
```

Mind being explicit with perms?
```suggestion
        platform-dev-flow:
          access_level: MANAGE_BUILD_AND_READ
```